src/xdocs/config_coding.xml

```diff
@@ -4468,6 +4468,31 @@ enum Foo10 { FOO, BAR } // OK
         <source>
 &lt;module name=&quot;NoFinalizer&quot;/&gt;
         </source>
+        <p>Example:</p>
+        <source>
+public class Test {
+
+    protected void finalize() throws Throwable { // violation
+        try {
+           System.out.println("overriding finalize()");
+        } catch (Throwable t) {
+           throw t;
+        } finally {
+           super.finalize();
+        }
+    }
+
+    protected void finalize(Object object) throws Throwable { // OK, overloaded method
+        try {
+           System.out.println("overriding finalizer() having parameters");
+        } catch (Throwable t){
+           throw t;
+        } finally {
+           super.finalize();
```

Comment:
The `object` parameter is missing here.

Reply:
I think it is fine, it is a call to original `finalize()`

Reply:
so, should i add it? and if yes, why?

Reply:
> I think it is fine, it is a call to original `finalize()`

A call to original `finalize()`? A quote from the Check description: `Finalizers are unpredictable, often dangerous, and generally unnecessary. Their use can cause erratic behavior, poor performance, and portability problems.`
The example should do not call the original `finalize`. And `finalize(object)` is not a finalizer. It is just a method with a poorly chosen name. It has nothing to do with GC.

Reply:
![image](https://user-images.githubusercontent.com/51538094/80309542-054e7380-87d6-11ea-83b4-10e213be2b0f.png)
i think this is a syntax error as this class is not inheriting 'finalizer(Object object)'

Reply:
> The example should do not call the original finalize

@pbludov , could you explain why? I thought the purpose of the whole `NoFinalizerCheck` check is to complain if you override the original `finalize` method (the first method above) but in the second method above you just call the original `finalize`. So, why there should not be a call to original `finalize` from another method?

Reply:
> i think this is a syntax error as this class is not inheriting 'finalizer(Object object)'

Why? `finalize` is not a reserved word. The example shows just a method with name `finalize`. This method is not called by GC.

**Maybe** the OK example should be just dropped, as it is already confusing people.
@rdiachenko, what are you think? The Check is not expected to have a **valid** case.

Reply:
If it confuses people I'm fine to remove this case. @OmarShawky1 could you please leave a `finalize` method with violation only?

Reply:
done